2016/05/08 03:24:00 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not allowed to access this page.' in /home/yii_framework/framework/gii/GiiModule.php:170
Stack trace:
#0 /home/yii_framework/framework/web/CController.php(263): GiiModule->beforeControllerAction(Object(DefaultController), Object(CInlineAction))
#1 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('')
#2 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii')
#3 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#4 /var/www/html/recipes/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/recipes/index.php?r=gii
---
2016/05/08 03:24:16 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not allowed to access this page.' in /home/yii_framework/framework/gii/GiiModule.php:170
Stack trace:
#0 /home/yii_framework/framework/web/CController.php(263): GiiModule->beforeControllerAction(Object(DefaultController), Object(CInlineAction))
#1 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('')
#2 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii')
#3 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#4 /var/www/html/recipes/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/recipes/index.php?r=gii
---
2016/05/08 03:24:19 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not allowed to access this page.' in /home/yii_framework/framework/gii/GiiModule.php:170
Stack trace:
#0 /home/yii_framework/framework/web/CController.php(263): GiiModule->beforeControllerAction(Object(DefaultController), Object(CInlineAction))
#1 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('')
#2 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii')
#3 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#4 /var/www/html/recipes/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/recipes/index.php?r=gii
---
2016/05/08 03:24:57 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not allowed to access this page.' in /home/yii_framework/framework/gii/GiiModule.php:170
Stack trace:
#0 /home/yii_framework/framework/web/CController.php(263): GiiModule->beforeControllerAction(Object(DefaultController), Object(CInlineAction))
#1 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('')
#2 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii')
#3 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#4 /var/www/html/recipes/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/recipes/index.php?r=gii
---
2016/05/08 03:25:00 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not allowed to access this page.' in /home/yii_framework/framework/gii/GiiModule.php:170
Stack trace:
#0 /home/yii_framework/framework/web/CController.php(263): GiiModule->beforeControllerAction(Object(DefaultController), Object(CInlineAction))
#1 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('')
#2 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii')
#3 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#4 /var/www/html/recipes/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/recipes/index.php?r=gii
---
2016/05/08 03:25:38 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not allowed to access this page.' in /home/yii_framework/framework/gii/GiiModule.php:170
Stack trace:
#0 /home/yii_framework/framework/web/CController.php(263): GiiModule->beforeControllerAction(Object(DefaultController), Object(CInlineAction))
#1 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('')
#2 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii')
#3 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#4 /var/www/html/recipes/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/recipes/index.php?r=gii
---
2016/05/08 03:25:58 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not allowed to access this page.' in /home/yii_framework/framework/gii/GiiModule.php:170
Stack trace:
#0 /home/yii_framework/framework/web/CController.php(263): GiiModule->beforeControllerAction(Object(DefaultController), Object(CInlineAction))
#1 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('')
#2 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii')
#3 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#4 /var/www/html/recipes/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/recipes/index.php?r=gii
---
2016/05/08 03:26:06 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not allowed to access this page.' in /home/yii_framework/framework/gii/GiiModule.php:170
Stack trace:
#0 /home/yii_framework/framework/web/CController.php(263): GiiModule->beforeControllerAction(Object(DefaultController), Object(CInlineAction))
#1 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('')
#2 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii')
#3 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#4 /var/www/html/recipes/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/recipes/index.php?r=gii
---
2016/05/08 03:27:05 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not allowed to access this page.' in /home/yii_framework/framework/gii/GiiModule.php:170
Stack trace:
#0 /home/yii_framework/framework/web/CController.php(263): GiiModule->beforeControllerAction(Object(DefaultController), Object(CInlineAction))
#1 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('')
#2 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii')
#3 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#4 /var/www/html/recipes/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/recipes/index.php?r=gii
---
2016/05/08 03:30:10 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /home/yii_framework/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/yii_framework/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/yii_framework/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/yii_framework/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/yii_framework/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/yii_framework/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/yii_framework/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/yii_framework/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/yii_framework/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/yii_framework/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/yii_framework/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/yii_framework/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/yii_framework/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /var/www/html/recipes/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/recipes/index.php?r=gii/model/index
HTTP_REFERER=http://128.199.129.162/recipes/index.php?r=gii/form/index
---
2016/05/08 03:38:31 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /home/yii_framework/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/yii_framework/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/yii_framework/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/yii_framework/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/yii_framework/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/yii_framework/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/yii_framework/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/yii_framework/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/yii_framework/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/yii_framework/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/yii_framework/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/yii_framework/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/yii_framework/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /var/www/html/recipes/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/recipes/index.php?r=gii/model/index
HTTP_REFERER=http://128.199.129.162/recipes/index.php?r=gii/form/index
---
2016/05/08 03:40:07 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /home/yii_framework/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/yii_framework/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/yii_framework/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/yii_framework/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/yii_framework/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/yii_framework/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/yii_framework/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/yii_framework/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/yii_framework/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/yii_framework/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/yii_framework/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/yii_framework/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/yii_framework/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /var/www/html/recipes/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/recipes/index.php?r=gii/model/index
HTTP_REFERER=http://128.199.129.162/recipes/index.php?r=gii/form/index
---
2016/05/08 03:40:08 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /home/yii_framework/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/yii_framework/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/yii_framework/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/yii_framework/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/yii_framework/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/yii_framework/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/yii_framework/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/yii_framework/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/yii_framework/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/yii_framework/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/yii_framework/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/yii_framework/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/yii_framework/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /var/www/html/recipes/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/recipes/index.php?r=gii/model/index
HTTP_REFERER=http://128.199.129.162/recipes/index.php?r=gii/form/index
---
2016/05/08 03:40:08 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /home/yii_framework/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/yii_framework/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/yii_framework/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/yii_framework/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/yii_framework/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/yii_framework/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/yii_framework/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/yii_framework/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/yii_framework/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/yii_framework/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/yii_framework/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/yii_framework/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/yii_framework/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /var/www/html/recipes/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/recipes/index.php?r=gii/model/index
HTTP_REFERER=http://128.199.129.162/recipes/index.php?r=gii/form/index
---
2016/05/08 03:40:09 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /home/yii_framework/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/yii_framework/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/yii_framework/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/yii_framework/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/yii_framework/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/yii_framework/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/yii_framework/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/yii_framework/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/yii_framework/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/yii_framework/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/yii_framework/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/yii_framework/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/yii_framework/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /var/www/html/recipes/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/recipes/index.php?r=gii/model/index
HTTP_REFERER=http://128.199.129.162/recipes/index.php?r=gii/form/index
---
2016/05/08 03:44:15 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /home/yii_framework/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/yii_framework/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/yii_framework/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/yii_framework/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/yii_framework/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/yii_framework/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/yii_framework/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/yii_framework/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/yii_framework/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/yii_framework/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/yii_framework/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/yii_framework/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/yii_framework/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /var/www/html/recipes/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/recipes/index.php?r=gii/model/index
HTTP_REFERER=http://128.199.129.162/recipes/index.php?r=gii/form/index
---
2016/05/08 03:44:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /home/yii_framework/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/yii_framework/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/yii_framework/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/yii_framework/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/yii_framework/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/yii_framework/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/yii_framework/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/yii_framework/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/yii_framework/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/yii_framework/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/yii_framework/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/yii_framework/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/yii_framework/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /var/www/html/recipes/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/recipes/index.php?r=gii/model/index
HTTP_REFERER=http://128.199.129.162/recipes/index.php?r=gii/form/index
---
2016/05/08 03:44:17 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /home/yii_framework/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/yii_framework/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/yii_framework/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/yii_framework/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/yii_framework/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/yii_framework/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/yii_framework/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/yii_framework/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/yii_framework/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/yii_framework/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/yii_framework/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/yii_framework/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/yii_framework/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /var/www/html/recipes/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/recipes/index.php?r=gii/model/index
HTTP_REFERER=http://128.199.129.162/recipes/index.php?r=gii/form/index
---
2016/05/08 03:45:18 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /home/yii_framework/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/yii_framework/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/yii_framework/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/yii_framework/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/yii_framework/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/yii_framework/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/yii_framework/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/yii_framework/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/yii_framework/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/yii_framework/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/yii_framework/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/yii_framework/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/yii_framework/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /var/www/html/recipes/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/recipes/index.php?r=gii/model/index
HTTP_REFERER=http://128.199.129.162/recipes/index.php?r=gii/form/index
---
2016/05/08 03:45:23 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /home/yii_framework/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/yii_framework/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/yii_framework/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/yii_framework/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/yii_framework/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/yii_framework/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/yii_framework/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/yii_framework/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/yii_framework/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/yii_framework/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/yii_framework/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/yii_framework/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/yii_framework/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /var/www/html/recipes/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/recipes/index.php?r=gii/model/index
HTTP_REFERER=http://128.199.129.162/recipes/index.php?r=gii/form/index
---
2016/05/08 03:48:26 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /home/yii_framework/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/yii_framework/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/yii_framework/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/yii_framework/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/yii_framework/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/yii_framework/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/yii_framework/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/yii_framework/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/yii_framework/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/yii_framework/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/yii_framework/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/yii_framework/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/yii_framework/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /var/www/html/recipes/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/recipes/index.php?r=gii/model/index
HTTP_REFERER=http://128.199.129.162/recipes/index.php?r=gii/form/index
---
2016/05/08 03:48:30 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /home/yii_framework/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/yii_framework/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/yii_framework/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/yii_framework/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/yii_framework/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/yii_framework/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/yii_framework/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/yii_framework/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/yii_framework/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/yii_framework/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/yii_framework/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/yii_framework/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/yii_framework/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/yii_framework/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/yii_framework/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/yii_framework/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /var/www/html/recipes/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/recipes/index.php?r=gii/model/index
---
